src/main/resources/user-list.txt.example

```diff
@@ -0,0 +1,2 @@
+# username
```

Comment:
is there really need to have two user-list admin-list example files with the same content?